app/src/main/java/org/wikipedia/page/linkpreview/LinkPreviewDialog.kt

```diff
@@ -227,14 +222,7 @@ class LinkPreviewDialog : ExtendedBottomSheetDialogFragment(), LinkPreviewErrorV
     }
 
     private fun doAddToList() {
-        addToListCallback.let {
-            if (it != null) {
-                it.onLinkPreviewAddToList(viewModel.pageTitle)
-            } else {
-                ExclusiveBottomSheetPresenter.showAddToListDialog(requireActivity().supportFragmentManager,
-                    viewModel.pageTitle, Constants.InvokeSource.LINK_PREVIEW_MENU)
-            }
-        }
+        ReadingListBehaviorsUtil.addToDefaultList(requireActivity(), viewModel.pageTitle, true, Constants.InvokeSource.LINK_PREVIEW_MENU)
```

Comment:
Should we call `dismiss()` here, so that the LinkPreviewDialog is dismissed and the snackbar can be seen?

Reply:
Done!